src/clack-utils.ts

```diff
@@ -81,7 +79,7 @@ export async function confirmContinueEvenThoughNoGitRepo(): Promise<void> {
     childProcess.execSync('git rev-parse --is-inside-work-tree', {
       stdio: 'ignore',
     });
-  } catch (e) {
+  } catch {
```

Comment:
Haha I now realized that optional catch binding was added in Node 10, so we can go through our SDK and update this also.

I guess the habit has stuck from working on the JS SDK codebase :)

Reply:
Yeah caught me too. Either ts or eslint in the SDK codebase disallow this though iirc.

Reply:
eslint in the codebase disallows empty block statements, but using optional catch is fine.

you just need to do

```js
catch {
  // plz eslint don't scream at me
}
```

It's es2019 though so no way we can do in the browser packages 😭